swagger/swagger-generator/generator-springmvc/src/main/java/org/apache/servicecomb/swagger/generator/springmvc/processor/annotation/RequestMappingMethodAnnotationProcessor.java

```diff
@@ -39,7 +41,9 @@ public void process(SwaggerGenerator swaggerGenerator, OperationGenerator operat
 
     doProcess(operationGenerator,
         requestMapping.path(), requestMapping.value(),
-        requestMapping.method().length == 0 ? null : requestMapping.method()[0],
+        requestMapping.method().length == 0 ?
+            (StringUtils.isEmpty(operationGenerator.getHttpMethod()) ? RequestMethod.GET : null) :
```

Comment:
Why use operationGenerator to get method not default to GET?

Reply:
take /swagger/swagger-generator/generator-springmvc/src/test/java/org/apache/servicecomb/swagger/generator/springmvc/Echo.java for example:
class level http method will set to operationGenerator,if method does not have http method,it will use class level http method,so only when both of class level and method level are not set,then we use GET as default.

@RequestMapping(
    path = "Echo",
    method = {RequestMethod.PUT},
    consumes = {"a", "b"},
    produces = {"a", "b"})
public class Echo {

  @RequestMapping
  public void emptyPath() {
  }

  @RequestMapping(method = {RequestMethod.GET, RequestMethod.POST})
  public void multiHttpMethod() {
  }

  @RequestMapping
  public void inheritHttpMethod(int query) {
  }
}